### ISSUE TITLE
[no-duplicate-imports]  false positive rule on "import type" 

### ISSUE THREAD

**Issue by zheeeng** [typescript-eslint/typescript-eslint#2315]
**Repro**

```ts
import type { Foo } from './foo'
import { foo } from './foo'
```

reports `eslint(no-duplicated-imports)` error

**Comment by karlismelderis** [comment]
Same problem here 👍 

rule is complaining when we do `import type` and `import` from same module

**Comment by mgol** [comment]
The doc at https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md says the replacement for the TSLint `no-duplicate-imports` rule is https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md, not the ESLint core rule.

**Comment by mgol-sumo** [comment]
@zheeeng BTW, this rule is named `no-duplicate-imports`, not `no-duplicated-imports`. Can you update the issue title & description with the correct name so that searching for this rule returns proper results?

**Comment by mgol** [comment]
@zheeeng BTW, this rule is named `no-duplicate-imports`, not `no-duplicated-imports`. Can you update the issue title & description with the correct name so that searching for this rule returns proper results?
